pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -1130,4 +1152,38 @@ private SuccessResponse updateZKTimeIntervalInternal(String tableNameWithType) {
       }
       return new SuccessResponse("Successfully updated time interval zk metadata for table: " + tableNameWithType);
   }
+
+  public List<Pair<TableType, List<String>>> selectSegments(
```

Comment:
Are we planning to add UTs for this method? If not, can we make this private?

Reply:
yeah, I forgot to change it to private